### PR TITLE
Create a mail merge using Gmail and Google Sheets

### PR DESCRIPTION
**Pull request by mhawksey**
Solution submission for issue ticket #54 "Create and distribute visual rich mail merges from data in Google Sheets" 

[If easier README can be commented on [in the Google Doc](https://docs.google.com/document/d/130XaDQShwcE83bYfFQi8rh6B5xkEIm_7qB-PiMsuaGo/edit?usp=sharing)]

### CODE REVIEW

File: mail-merge/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: Create a mail merge using Gmail and Google Sheets
+description: Create and distribute visual rich mail merges with Gmail and Google Sheets.
```
Comment (sqrrrl) [inline mail-merge/README.md]: visual -> visually

File: mail-merge/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+* Uses the [Gmail Service](https://developers.google.com/apps-script/reference/gmail/) to read drafts and to send emails.
+
+* Uses  [SpreadsheetApp service](https://developers.google.com/apps-script/reference/spreadsheet/spreadsheet-app) to read and write data.
+
+## Try it
+
+1. Create a copy of the sample [Gmail/Sheets Mail Merge spreadsheet](https://docs.google.com/spreadsheets/d/1EfjLuYGab8Xt8wCn4IokBIG0_W4tBtiU4vxl3Y7FPsA/copy).
+
+2. Update the ‘Recipients’ column with email addresses you would like to use in the mail merge
```
Comment (sqrrrl) [inline mail-merge/README.md]: Use bold instead of quotes for Recipients.

File: mail-merge/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+3. Create a draft message in your Gmail account using markers like {{First name}}, which correspond to column names, to indicate text you’d like to be replaced with data from the copied spreadsheet. 
+
+4. Click on custom menu item **Mail Merge > Send Emails**.
+
+5. A dialog box will appear and tell you that the script requires authorization. Read the authorization notice and continue.
+
+6. When prompted enter or copy/paste the subject line used in your draft Gmail message and click ‘OK’
```
Comment (sqrrrl) [inline mail-merge/README.md]: Use bold instead of quotes for OK.

File: mail-merge/README.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+* Uses  [SpreadsheetApp service](https://developers.google.com/apps-script/reference/spreadsheet/spreadsheet-app) to read and write data.
+
+## Try it
+
+1. Create a copy of the sample [Gmail/Sheets Mail Merge spreadsheet](https://docs.google.com/spreadsheets/d/1EfjLuYGab8Xt8wCn4IokBIG0_W4tBtiU4vxl3Y7FPsA/copy).
+
+2. Update the ‘Recipients’ column with email addresses you would like to use in the mail merge
+
+3. Create a draft message in your Gmail account using markers like {{First name}}, which correspond to column names, to indicate text you’d like to be replaced with data from the copied spreadsheet.
```
Comment (sqrrrl) [inline mail-merge/README.md]: suggest putting {{First name}} in code font (backticks)

File: mail-merge/README.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+3. Create a draft message in your Gmail account using markers like {{First name}}, which correspond to column names, to indicate text you’d like to be replaced with data from the copied spreadsheet. 
+
+4. Click on custom menu item **Mail Merge > Send Emails**.
+
+5. A dialog box will appear and tell you that the script requires authorization. Read the authorization notice and continue.
+
+6. When prompted enter or copy/paste the subject line used in your draft Gmail message and click ‘OK’
+
+7. The ‘Email Sent’ column will update with the message status.
```
Comment (sqrrrl) [inline mail-merge/README.md]: Use bold for 'Email Sent'

File: mail-merge/README.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+## Next steps
+
+Additional columns can be added to the spreadsheet with other data you would like to use. Using the {{}} annotation and including your column name as part of your Gmail draft will allow you to include other data from your spreadsheet. If you change the name of the ‘Recipient’ or ‘Email Sent’ columns this will need to be updated by opening **Tools > Script Editor**.
```
Comment (sqrrrl) [inline mail-merge/README.md]: Nits:
{{}} in code font
Recipient & Email Sent in bold

**Comment by mhawksey** [comment]
@sqrrrl thanks for the feedback. Hopefully I've picked up all those suggested edits now. Let me know if there is anything else

**Comment by sqrrrl** [comment]
Looks great :)

**Pull request merged**